bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/async/TestAsyncSnapshots.java

```diff
@@ -162,4 +163,13 @@ protected void deleteTable(TableName tableName) throws IOException {
       throw new IOException("Error while deleting table: "+e.getCause());
     }
   }
+
+  @Override
+  protected int listTableSnapshotsSize(Pattern tableNamePattern) throws Exception {
+	try {
```

Comment:
please fix the whitespace.

Reply:
Hi Solomon, Thanks for the review, Have removed white-spaces from all the files.